### PR TITLE
Introduction to PHP Operators

### LINKED ISSUE #2545: Introduction to PHP Operators
### Proposed title of article
Introduction to PHP Operators

### Introduction paragraph (2-3 paragraphs):
PHP Operator is a symbol i.e used to perform operations on operands. In simple words, operators are used to perform operations on variables or values. For example:

$num=10+20;//+ is the operator and 10,20 are operands  

In PHP, we have different types of operators depending on the number of operands they take. There are some operators that take only one operand and perform an operation on it. This type of operators are called unary operators. Another type of operators are binary operators which take two operands.

### Key takeaways: 
1. Instaling PHP and Xampp in Linux
2. Define what is an operator and an operand
3. Learn the roles of different operators
4. Category of PHP Operators
5. Use Operators to perform mathmatical computation

### References: 
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by mussooo**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
This fixes  #2545

### CODE REVIEW

**Comment by hectorkambow** [comment]
@jerimkaura  Will be reviewing this article 👍 

Ping with any questions.

**Comment by jerimkaura** [comment]
Hello @mussooo, welcome to Engineering Education Program. Thank you for submitting your first article for review. I will be your reviewer to check on technical and grammatical errors just in case there are any before your article is published.

File: content/articles/introduction-to-php-operators/index.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+```
+Hello World! 
+```
+B. Following `concatenation assignment operator` we have the following Example:
+```php
+$a = "Good ";
+$a .= "Work!"; // here, $a contains "Good Work!"
+echo "$a";
+?>
+```
```
Comment (jerimkaura) [inline content/articles/introduction-to-php-operators/index.md:225]:
```suggestion
The `concatenation operator` ('.')  which returns the right and left arguments concatenated and the  `concatenation assignment operator`  ('.=')  which appends the right-hand argument to the left-hand argument.

- Concatenation Operator.  The example below demonstrates the use of a `concatenation operator` :
```php
$a = "Hello ";
$b = $a . "World!"; // here, $b contains "Hello World!"
echo "$b";
?>
```
Output
```
Hello World! 
```
- Concatenation Assignment Operator.  The example below demonstrates the use of a `concatenation assignment operator` :
```php
$a = "Good ";
$a .= "Work!"; // here, $a contains "Good Work!"
echo "$a";
?>
```
```

File: content/articles/introduction-to-php-operators/index.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+There are various Arithmetic operators supported by PHP:
+
+1. Addition Operator(+)  - They add two operands.
+
+2. Subtraction Operator(-) - Subtract the second operand from the first operand.
+
+3. Muitiplacation Operators (*) - These multiply two operands.
+
+4. Division Operator (/) - Divide numerator by de-numerator
+
```
Comment (jerimkaura) [inline content/articles/introduction-to-php-operators/index.md:88]:
```suggestion
These are operators that are used on the operator to execute arithmetic operations.

There are various Arithmetic operators supported by PHP:

1. Addition Operator(+)  - It adds two operands.

2. Subtraction Operator(-) - Subtracts the second operand from the first operand.

3. Muitiplacation Operators (*) - These multiplies two operands.

4. Division Operator (/) - Divides numerator by denominator

```

**Review by jerimkaura — CHANGES_REQUESTED** [review]
@mussooo Kindly go through the suggestions I have made. Ping me when you are done so that we can proceed to tech review.

cc @hectorkambow

**Comment by jerimkaura** [comment]
The hero image for your article should be approximately 600x400 pixels. The one you provided was way too smaller. Be sure to use a suitable image instead .

**Comment by jerimkaura** [comment]
@mussooo, Thank you making the changes, let me go through it before we proceed to tech review.

**Comment by hectorkambow** [comment]
@jerimkaura is doing a test review on this PR 👍 

cc @louisefindlay23 @ahmadmardeni1 @LinusMuema can answer any questions that pop up - and changes the labels when needed.

File: content/articles/introduction-to-php-operators/index.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+### Prerequisites
+
+1.  Xampp for `Linux` downloaded on your machine. You can download the latest version [here](https://www.apachefriends.org/download.html) or follow [this]() steps on how to download.
+2. Little knowledge of PHP.
+3.  A text editor like [Visual studio](https://code.visualstudio.com/download) or any other depending on your taste.
+
+`NB`
+To run any PHP file, You must first start `Xampp`. This is done using this command.
+```bash
```
Comment (jerimkaura) [inline content/articles/introduction-to-php-operators/index.md:12]:
```suggestion
3.  A text editor like [Visual studio](https://code.visualstudio.com/download) or any other depending on your taste.
>To run any PHP file, You must first start `Xampp`. This is done using this command.
```bash
```

File: content/articles/introduction-to-php-operators/index.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
+### Introduction
+An operator is is a symbol that performs operations on operands. In [PHP(Hypertext Preprocessor)](https://www.php.net/), there are different types of operators depending on the number of operands they take. For example, urinary operators take only one operand. On the contrary, binary operators take two operands..
+
+### Prerequisites
+
+1.  Xampp for `Linux` downloaded on your machine. You can download the latest version [here](https://www.apachefriends.org/download.html) or follow [this]() steps on how to download.
```
Comment (jerimkaura) [inline content/articles/introduction-to-php-operators/index.md:6]:
```suggestion
### Prerequisites
1.  Xampp for `Linux` downloaded on your machine. You can download the latest version [here](https://www.apachefriends.org/download.html) or follow [this]() steps on how to download.
```

**Review by jerimkaura — CHANGES_REQUESTED** [review]
@mussooo , You must have forgotten to remove extra spaces from your article. Kindly finish on that then we proceeded to tech review.

**Comment by mussooo** [comment]
> @mussooo , You must have forgotten to remove extra spaces from your article. Kindly finish on that then we proceeded to tech review.

@jerimkaura I had an issue with my laptop that why I did not complete removing all the empty spaces.

**Comment by mussooo** [comment]
@jerimkaura Please check my last commit.

**Comment by mussooo** [comment]
@jerimkaura changes done. Please have a look at my last commit.

**Comment by jerimkaura** [comment]
Amazing🤝, @mussooo. I  will begin the tech review right away. In this phase, I will go through your code snippets to ensure that they actually work perfectly and have empirical accuracy.

**Comment by jerimkaura** [comment]
@kindly separate each code snippet with its output to facilitate understanding by beginners.

**Comment by mussooo** [comment]
@jerimkaura changes done. Please have a look.

**Comment by jerimkaura** [comment]
@mussooo you did not make the changes. You only fetched and merged the master branch. The changes should be done in the `index.md` file on the `mussooo:introduction-to-php-operators` branch.

**Comment by mussooo** [comment]
Hey @jerimkaura, I've opened a new PR. You may proceed with the review. Thank you!

**Comment by jerimkaura** [comment]
Hello @hectorkambow , @ahmadmardeni1 the students opened a new PR here #2983 having had a merge conflict due to the [907d0ab](https://github.com/section-engineering-education/engineering-education/pull/2895/commits/907d0aba51e2d7f444492858579cd9eacb39b4ae) commit. here is the new PR #2983

**Pull request closed without merging**